Functions/Statistics/xASL_stat_PrintStats.m

```diff
@@ -153,7 +155,6 @@
             
             if isempty(startSessionIndex) || isempty(endSessionIndex)
 				% If the sessionID was not found, then report a warning and use the entire expression for subjectID
-				SubjectID = x.S.SubjectSessionID{iSubjSess}(startSubjectIndex:endSubjectIndex);
```

Comment:
I considered it, bu I removed this line because the variable created here (SubjectID) wasn't ever used anywhere :)

Reply:
I thought I saw that it is now also created elsewhere. Are you really sure? If it doesn't hurt, I would just comment it out instead of removing it; as I cannot oversee now how this works when doing rc1T1 stats. Easier for me when I'm going to improve the voxel-wise stats

Reply:
Not the exact same parameter. I full-text search the entire file to always make sure it's nowhere used, saved, or passed as an output. So it was really useless. I comment out things that can be potentially used in the future, but this didn't seem to be the case ;) Fine to keep it like this?

Reply:
I would comment it out rather than delete it here, just because I don't know how this affects the usage for images that don't have sessions like rc1T1. We can also test this use case but easier to just move it back in and comment out? Up to you.